http://bugs.freedesktop.org/show_bug.cgi?id=62220

Failure to compile on SH4 arch because <sys/wait.h> is missing for #define's of:
WIFEXITED, WEXITSTATUS, WIFSIGNALED, WTERMSIG

Errors are like this:

polkitbackendjsauthority.c:1509:3: warning: implicit declaration of function 'WIFEXITED' [-Wimplicit-function-declaration]
polkitbackendjsauthority.c:1509:3: warning: nested extern declaration of 'WIFEXITED' [-Wnested-externs]

polkitbackendjsauthority.c:(.text+0x3008): undefined reference to `WIFEXITED'
polkitbackendjsauthority.c:(.text+0x3010): undefined reference to `WIFEXITED'

--- src/polkitbackend/polkitbackendjsauthority.c
+++ src/polkitbackend/polkitbackendjsauthority.c
@@ -20,6 +20,7 @@
  */
 
 #include "config.h"
+#include <sys/wait.h>
 #include <errno.h>
 #include <pwd.h>
 #include <grp.h>
